BookingQueueSubscriber/BookingQueueSubscriber.Services/NotificationApi/AddNotificationRequestMapper.cs

```diff
@@ -7,7 +7,8 @@ namespace BookingQueueSubscriber.Services.NotificationApi
 {
     public static class AddNotificationRequestMapper
     {
-        public static AddNotificationRequest MapToNewUserNotification(Guid hearingId, ParticipantDto participant, string password)
+        public static AddNotificationRequest MapToNewUserNotification(Guid hearingId, ParticipantDto participant,
```

Comment:
Isn't the `AddNotificationRequest` now obsolete. There are endpoints specific to each of these scenarios

<img width="1499" alt="image" src="https://github.com/hmcts/vh-booking-queue-subscriber/assets/41630528/3e68bf40-485f-47c3-b7a5-a8ff200ac065">

Reply:
Notification API handles the concerns re which template to use. Please point to the non-obsolete endpoints to avoid breaking the flow.

<img width="563" alt="image" src="https://github.com/hmcts/vh-booking-queue-subscriber/assets/41630528/d33bfce3-a37f-4cb8-8749-bf3a94f1df23">

<img width="1027" alt="image" src="https://github.com/hmcts/vh-booking-queue-subscriber/assets/41630528/d7d983b1-a18b-4122-86f1-a5689e7bfa8c">

Reply:
the plan is to remove the code only for the feauture toggle. We decided to leave the endpoint and delete them with an other task

Reply:
I am not sure that this should be part of this ticket

Reply:
that's fair. I will raise a new ticket to move away from the obsolete endpoints